library.properties

```diff
@@ -0,0 +1,9 @@
+name=I-NUCLEO-LRWAN1

```

Comment:
Add stm32duino in the name to be aligned with other libraries

Reply:
name=STM32duino I-NUCLEO-LRWAN1